src/style.css

```diff
@@ -1 +1,83 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir.ttc');
+  font-weight: bold;
```

Comment:
Consider adding a fallback font family in your `@font-face` rule to ensure text remains visible when the custom font is loading or if it fails to load. For example, you could modify the `font-family` property to include a generic font family like `sans-serif` as a backup.